Packs/ReversingLabs_Titanium_Cloud/Integrations/ReversingLabsTitaniumCloudv2/ReversingLabsTitaniumCloudv2.py

```diff
@@ -0,0 +1,920 @@
+from typing import Union
+import demistomock as demisto
+from CommonServerPython import *
+from ReversingLabs.SDK.ticloud import FileReputation, AVScanners, FileAnalysis, RHA1FunctionalSimilarity, \
+    RHA1Analytics, URIStatistics, URIIndex, AdvancedSearch, ExpressionSearch, FileDownload, FileUpload, \
+    URLThreatIntelligence, AnalyzeURL, DynamicAnalysis, CertificateAnalytics
+
+
+VERSION = "v2.0.0"
+USER_AGENT = f"ReversingLabs XSOAR TitaniumCloud {VERSION}"
+
+TICLOUD_URL = demisto.params().get("base")
+USERNAME = demisto.params().get("credentials", {}).get("identifier")
+PASSWORD = demisto.params().get("credentials", {}).get("password")
+RELIABILITY = demisto.params().get("reliability", "C - Fairly reliable")
+
+
+def classification_to_score(classification):
+    score_dict = {
+        "UNKNOWN": 0,
+        "KNOWN": 1,
+        "SUSPICIOUS": 2,
+        "MALICIOUS": 3
+    }
+    return score_dict.get(classification, 0)
+
+
+def test_module_command():
+    mwp = FileReputation(
+        host=TICLOUD_URL,
+        username=USERNAME,
+        password=PASSWORD
+    )
+
+    try:
+        _ = mwp.get_file_reputation(hash_input="6a95d3d00267c9fd80bd42122738e726")
+    except Exception as e:
+        return_error(str(e))
+
+    result = "ok"
+    return_results(result)
+
+
+def file_reputation_command():
+    mwp = FileReputation(
+        host=TICLOUD_URL,
+        username=USERNAME,
+        password=PASSWORD
+    )
+
+    hash_value = demisto.getArg("hash")
+
+    try:
+        response = mwp.get_file_reputation(hash_input=hash_value)
+    except Exception as e:
+        return_error(str(e))
+
+    response_json = response.json()
+
+    results = file_reputation_output(response_json=response_json, hash_value=hash_value)
+    return_results(results)
+
+
+def file_reputation_output(response_json, hash_value):
+    malware_presence = response_json.get("rl", {}).get("malware_presence")
+    if not malware_presence:
+        return_error("There is no malware_presence object in the response JSON.")
+
+    classification = malware_presence.get("status")
```

Comment:
There is always a "status" key in the report if there is any report at all. The script would otherwise fail earlier. Also, this particular API always returns a JSON body. If there file is not known, the "status" will be "UNKNOWN".